_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,130 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can dispatch a variety of messages through various channels
+and varying formats alerting you to events in your environment.
+
+### Notification Types
+
+  * Inventory Summary: Count of the resources added in the latest inventory crawl.
+  * Violation Notifications: Individual violations that have been found from the latest scanner run. 
+  * Cloud SCC Findings: Violations converted to Cloud SCC (Cloud Security Command Center) findings format, that's ingestable by Cloud SCC.
+  
+### Notification Channels
+
+  * Email
+  * Slack
+  * Cloud Storage
+
+### Notification Formats
+
+  * Human-readable data: CSV
+  * Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary
+
+This is a count of what resources have been crawled into inventory,
+and outputted to Cloud Storage bucket.
+
```

Comment:
You should introduce the sample configuration in some way by describing what's in it and what it's doing.